easybuild/easyconfigs/t/torchvision/torchvision-0.16.2-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,27 @@
+name = 'torchvision'
```

Comment:
I think it would be better to start using the PyTorch-bundle instead of updating these individual packages. It's a lot cleaner from a packaging situation given how often we update PyTorch within a toolchain release.

Reply:
forgot that was a thing

Reply:
@Micket PR updated